images/php/cli/Dockerfile

```diff
@@ -74,5 +74,8 @@ COPY id_ed25519_lagoon_cli.key /home/.ssh/lagoon_cli.key
 RUN chmod 400 /home/.ssh/lagoon_cli.key
 ENV SSH_AUTH_SOCK=/tmp/ssh-agent
 
+# readinessProbe check
+COPY 999-readiness.sh /lagoon/entrypoints/
```

Comment:
Let's add this to the `commons` image where it will then end up in all docker images from lagoon, this solves two issues:
1. we can use the check for `/tmp/ready` also in other places
2. there might be some people that use the `cli` services with another base image than the php-cli Dockerimage, for these people I would at least hope that they use the lagoon commons image as a base and therefore they will also end up having this check inside of them.